src/js/components/CheckBox/CheckBox.js

```diff
@@ -194,6 +195,7 @@ const CheckBox = forwardRef(
     return (
       <StyledCheckBoxContainer
         aria-label={a11yTitle}
+        fillProp={fill}
```

Comment:
```suggestion
        fill={fill}
```
Can we just pass through as `fill`? If so, would also need to update the StyledCheckBox stuff

Reply:
oh yes you just pass in `fill`